packages/react-liveness/src/components/FaceLivenessDetector/service/utils/getFaceMatchState.ts

```diff
@@ -0,0 +1,117 @@
+import {
+  LivenessOvalDetails,
+  Face,
+  FaceMatchState,
+  BoundingBox,
+} from '../types';
+import { SessionInformation } from '@aws-sdk/client-rekognitionstreaming';
+import {
+  generateBboxFromLandmarks,
+  getIntersectionOverUnion,
+  getOvalBoundingBox,
+} from './liveness';
+
+/**
+ * Returns the state of the provided face with respect to the provided liveness oval.
+ */
+// eslint-disable-next-line max-params
```

Comment:
1. We should update the signature of `getFaceMatchStateInLivenessOval` to be an object so we can remove this `eslint-disable`
2. Let's rename the function to `getFaceMatchState` to align with the file name